lib/client_transaction_id.rb

```diff
@@ -1,9 +1,11 @@
 module ClientTransactionId
-  attr_accessor :cl_trid_prefix, :cl_trid
+  attr_accessor :cl_trid_prefix, :cl_trid, :schema_version, :schema_prefix
 
   def initialize(args = {})
     self.cl_trid = args[:cl_trid]
     self.cl_trid_prefix = args[:cl_trid_prefix]
+    @schema_version = args[:schema_version]
```

Comment:
Do we have any default values or 'nil' arguments will break the gem?
If so - we are breaking backward comparability and we need to raise major gem version.